src/layout/Layout.styles.js

```diff
@@ -326,7 +326,7 @@ export const GlobalStyle = createGlobalStyle`
   a:not([class]) {
     text-decoration: none;
     border-bottom: 2px solid ${colors.eucalyptusGreen};
-    color: ${colors.black};
+    color: ${colors.indigo};
```

Comment:
This will change the global appearance for any links in the site that use this style. For example, if you look at your `/about-us` page (and compare it to the live site), you'll see it's updated the links in there to be indigo. The styles for the links should only be altered within the blog context for now.

Reply:
Oh yes, you're right! Thanks for pointing it out. This was the only file where a change to `indigo` resulted in a UI change though. I'll dig around a bit more to find other options.

Reply:
Fixed it now :)